cfme/tests/intelligence/reports/test_menus.py

```diff
@@ -171,3 +169,58 @@ def test_rbac_move_custom_report(
             type=folder, subtype=subfolder, menu_name=get_custom_report.menu_name
         )
         assert rbac_report.exists
+
+
+@pytest.mark.tier(1)
+@pytest.mark.ignore_stream("5.10")
+@pytest.mark.meta(automates=[1676638, 1731017])
+@pytest.mark.parametrize("group", GROUPS)
+def test_reports_menu_with_duplicate_reports(appliance, request, group, report_menus):
+    """
+    Polarion:
+        assignee: pvala
+        casecomponent: Reporting
+        initialEstimate: 1/10h
+        startsin: 5.9
+        setup:
+            1.Create a custom report or copy an existing report.
+            2. Move the custom report to 'Tenants > Tenant Quotas' menu in user's current group
+        testSteps:
+            1. See if the report is available under 'Tenants > Tenant Quotas'.
+            2. Delete the custom report.
+            3. See if the report is available under 'Tenants > Tenant Quotas'.
+            4. Add a new report with the same menu_name.
+            5. See if the report is available under 'Tenants > Tenant Quotas'.
+        expectedResults:
+            1. Report must be visible.
+            2.
+            3. Report must not be visible.
+            4.
+            5. Report must not be visible.
+    Bugzilla:
+        1731017
+        1676638
+    """
```

Comment:
@valaparthvi 
**Optional:**
- It will be better if you put ```Bugzilla``` block above ```Polarion``` data.
- In my observation, Bugzilla numbers are not getting updated in polarion you keep it as it is.
Please check your old polarion test cases once and decide.